modules/serverless-web-console.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * layered-products-web-console.adc
+// * products-web-console.adoc
```

Comment:
Hi @kaldesai, @rh-max @pierDipi, on line 9 in this file we have a short write-up about Serverless on webconsole. 
IMHO, we should probably highlight that - Serverless provides developers with a modern, cloud-native app dev stack for hybrid clouds or enables developers to create and deploy serverless, event-driven applications on OpenShift Container Platform as the key or the first point to highlight. 
The singles vendor/multicloud approach could be the second sentence. WDYT?

Reply:
@Preeticp @pierDipi and @rh-max +1 to the suggestion. We can use the overview description as well and try to align the content with the web console description. 

Maybe something like this: 

`OpenShift Serverless allows developers to create and deploy serverless, event-driven applications on the OpenShift Container Platform. It provides portability and consistency for hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform. Using Red Hat OpenShift Serverless allows you to avoid being locked into a single vendor and enables a multi-cloud approach. You can use the OpenShift Container Platform web console OperatorHub to install the OpenShift Serverless Operator. The Operator enables you to perform actions such as view event source types, examine metrics, create a function, and more.`

Reply:
I would like to keep these descriptions brief. I also think this should just be a web console focused description given that it is in the web console section of the docs. If a user wants to know more about serverless and they are in the OCP documentation, they are going to go to the serverless section and get redirected to the serverless docs.
The intention of this is to give people a quick snapshot, let them know they can use it in the console, and redirect people to the location they can get more information. 
I will revise the wording of the first sentence a bit.

Reply:
@opayne1 if we want to capture only one sentence for a short paragraph on what Serverless does on the webconsole, I suggest: 'Serverless enables developers to create and deploy serverless, event-driven applications on OpenShift Container Platform.'
That captures the essence of what Serverless provides on the web console too. 
The rest of is covered on RH.com/in the prod docs as you mention.

Reply:
Updated. Thanks you all for the input.